lib/ibm/helpers/connection.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'time'
+require 'http'
+
+module IBM
+  module Cloud
+    module SDK
+      # Contols tokens.
+      class Connection
```

Comment:
Should this be in `lib/ibm/cloud/sdk/connection.rb` ?

Reply:
Yes, I am still playing with this portion trying to get it right. I'm doing two things in here.
1) Changing from rest-client to http. http looks to be faster than rest-client. https://github.com/httprb/http and in my benchmarks this bears out sort of.
2) Have the token refresh automatically.